<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>883</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>华为天线提取软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_import">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="image_import"/>
    </widget>
    <widget class="QMenu" name="menu_recentFiles">
     <property name="title">
      <string>最近打开文件</string>
     </property>
    </widget>
    <addaction name="createAct"/>
    <addaction name="openAct"/>
    <addaction name="menu_recentFiles"/>
    <addaction name="separator"/>
    <addaction name="closeAct"/>
    <addaction name="saveAct"/>
    <addaction name="saveAsAct"/>
    <addaction name="separator"/>
    <addaction name="menu_import"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="toolMenu">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <widget class="QMenu" name="segmentMenu">
     <property name="title">
      <string>元素分割</string>
     </property>
     <addaction name="actionLazyBrush"/>
     <addaction name="actionGrabCut"/>
     <addaction name="actionGraphCut"/>
     <addaction name="actionMixture"/>
    </widget>
    <addaction name="segmentMenu"/>
    <addaction name="action_mix"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="helpAct"/>
    <addaction name="aboutAct"/>
   </widget>
   <widget class="QMenu" name="editMenu">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="undoAct"/>
    <addaction name="redoAct"/>
    <addaction name="separator"/>
    <addaction name="cutAct"/>
    <addaction name="copyAct"/>
    <addaction name="pastAct"/>
    <addaction name="separator"/>
    <addaction name="optionAct"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="zoom_in"/>
    <addaction name="zoom_out"/>
    <addaction name="viewInScreenAct"/>
    <addaction name="viewInActualPixAct"/>
   </widget>
   <widget class="QMenu" name="windowMenu">
    <property name="title">
     <string>窗口(&amp;W)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="separator"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="separator"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="editMenu"/>
   <addaction name="toolMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="windowMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>工具栏</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="createAct">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="openAct">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="closeAct">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="closeAllAct">
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="text">
    <string>存储</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="exitAct">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="helpAct">
   <property name="text">
    <string>帮助主题</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="aboutAct">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="undoAct">
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="redoAct">
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="optionAct">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="text">
    <string>关闭所有</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>划分窗口</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>重叠窗口</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>下一个窗口</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>上一个窗口</string>
   </property>
  </action>
  <action name="saveAsAct">
   <property name="text">
    <string>存储为</string>
   </property>
  </action>
  <action name="image_import">
   <property name="text">
    <string>原始拍摄图像</string>
   </property>
  </action>
  <action name="cutAct">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="copyAct">
   <property name="text">
    <string>拷贝</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="pastAct">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="zoom_in">
   <property name="text">
    <string>放大</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="zoom_out">
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="viewInScreenAct">
   <property name="text">
    <string>适合屏幕</string>
   </property>
  </action>
  <action name="viewInActualPixAct">
   <property name="text">
    <string>实际像素</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionPanorama">
   <property name="text">
    <string>拼接</string>
   </property>
  </action>
  <action name="actionElementQuery">
   <property name="text">
    <string>元素检索</string>
   </property>
  </action>
  <action name="actionLazyBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lazybrush</string>
   </property>
  </action>
  <action name="actionGrabCut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Grabcut</string>
   </property>
  </action>
  <action name="actionGraphCut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Graphcut</string>
   </property>
  </action>
  <action name="actionMixture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mixture</string>
   </property>
  </action>
  <action name="actionAddElement">
   <property name="text">
    <string>添加到数据库</string>
   </property>
  </action>
  <action name="actionOpenBigImage">
   <property name="text">
    <string>打开bigimage文件</string>
   </property>
  </action>
  <action name="actionAddElementToPrj">
   <property name="text">
    <string>添加到工程</string>
   </property>
  </action>
  <action name="actionSaveInpainting">
   <property name="text">
    <string>保存修复图像</string>
   </property>
  </action>
  <action name="action_getmask">
   <property name="text">
    <string>导出mask</string>
   </property>
  </action>
  <action name="action_getFront">
   <property name="text">
    <string>导出前景</string>
   </property>
  </action>
  <action name="action_mix">
   <property name="text">
    <string>合并图像</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
